File: todo_back/src/main/java/com/example/todo_back/domain/UserInfo.java
package com.example.todo_back.domain;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
public class UserInfo {
    @Id
    private Long id;

    private String profileNickname;

    private String profileImage;

    private String accountEmail;
}

File: todo_front/lib/data/models/user_info.dart
class UserInfo {
  final int id;
  final String profileNickname;
  final String profileImage;
  final String accountEmail;

  UserInfo({
    required this.id,
    required this.profileNickname,
    required this.profileImage,
    required this.accountEmail,
  });

  factory UserInfo.fromMap(Map userInfoMap) {
    return UserInfo(
      id: userInfoMap['id'],
      profileNickname: userInfoMap['profileNickname'],
      profileImage: userInfoMap['profileImage'],
      accountEmail: userInfoMap['accountEmail'],
    );
  }
}
